test-cases/gemini/gemini-3h-ics-cdc-with-nemesis.yaml

```diff
@@ -30,4 +30,4 @@ gemini_table_options:
 stress_cdclog_reader_cmd: "cdc-stressor -duration 215m -stream-query-round-duration 30s"
 
 db_type: mixed_scylla
-instance_type_db_oracle: 'i3.8xlarge'
+instance_type_db_oracle: 'i4i.8xlarge'
```

Comment:
this was a problem previously, since we had oracle versions which don't support i4i (2021.1, I think)
this needs testing

Reply:
Of course. Everything needs testing.
Partially related - we should upgrade oracle at some point...

Reply:
@roydahan update it last in #6988

we probably need a better definition when those need to change, and not only when things breaks (or AMIs get deleted)

Reply:
I think we should update it on every major Scylla release. We continue to fix bugs, even in data consistency. I don't know if the Oracle has cache disabled, for example (I hope!), but we've had some rare bugs there - with RF=1 exposing them, etc.

Reply:
It is using the following in defaults to disable cache:
append_scylla_args_oracle: '--enable-cache false'

The assumption is that old releases are good and tested and working for customers and new releases may regress (which is mostly what happens).

Anyway, i4i should work for every release we have here.